docs-team/2017-05/may-25.md

```diff
@@ -0,0 +1,103 @@
+# Learn Team Agenda 2017/05/25
+
+## Topics
+
+- Infrastructure/Heroku (TL, SK)
+  - Fastly setup (SK)
+  - Heroku CI
+  - Heroku Previews
+- Algolia search (MS)
+- Route restructuring (MS)
+- Fastboot support (http://bit.ly/2qnUOHi) (MS)
+- `<version-select></version-select>` (JW)
+- Add docs generation to ember & ember-data build pipelines (SK)
+
+## Action Items
+
+[ ] RM: Will experiment with statically building Guides with Ember @Ricardo M
+[ ] DB: Will keep working on reworking Middleman build process
+[ ] MS: Will finish route restructuring @Michael S
+[ ] MS: Will research Algolia setup for beta search @Michael S
+
+## Attendees (Initials: Name @slack-id)
+
+- TJ: Todd Jordan @todd.jordan
+- RM: Ricardo Mendes @locks
+- MS: Michael Schinis @mschinis
+- TL: Terence Lee @hone
+- SK: Sivakumar Kailasam @siva
+- YB: Yaw Boakye @yaw
+
+## Notes
+
+
+### API Docs Update
+
+MS: Working on route restructuring PR for Algolia search.  Started on it and should have something for review by next Tuesday: https://github.com/ember-learn/ember-api-docs/pull/224
+
+MS: Also there’s an issue for us with the release candidate of fastboot that breaks stuff.  They’ve taken away instance initializers which we rely on.  There is working going on to restore but we may want to depend on the previous release (beta 16 or 15) for the time being.
```

Comment:
You could depend on any non-rc build. `beta.19` which is the latest should also work.